docs/faq/index.md

```diff
@@ -0,0 +1,93 @@
+# Frequently asked questions
+
+## General questions
+
+### What does the GitOps Toolkit mean for Flux?
+
+Flux v1 is a monolithic do-it-all operator, the GitOps Toolkit separates the functionalities into specialized controllers.
+
+Flux v2 will be a curated set of configuration for the GitOps Toolkit which you can install and operate simply using the `tk` command. You can easily pick and choose the functionality you need and add extensions to enhance Flux functionality.
+
+The timeline we are looking at right now is:
+
+1. Put Flux v1 into maintenance mode (no new features being added).
+1. Continue work on [GitOps Toolkit roadmap](https://toolkit.fluxcd.io/roadmap/).
+1. We will provide transition guides for specific user groups, e.g. users of Flux v1 in read-only mode, or of Helm Operator v1, etc. once the functionality is integrated in the GitOps Toolkit and it's deemed "ready".
+1. Once the use-cases of Flux v1 are covered, we will continue supporting Flux v1 for 6 months. This will be the transition period before it will reach it's end of life.
```

Comment:
```suggestion
1. Once the use-cases of Flux v1 are covered, we will continue supporting Flux v1 for 6 months. This will be the transition period before it's considered unsupported.
```